.gitignore

```diff
@@ -1,7 +1,7 @@
 go-f2lb-website
 
 vendor
-
+node_modules
```

Comment:
nodejs related files should be not in the top dir but only under webui, also files like package.json and yarn.lock are meaningful to be tracked under git.

Reply:
Sorry this shouldn't be in PR